decidim-meetings/app/cells/decidim/meetings/highlighted_meetings_for_component_cell.rb

```diff
@@ -81,6 +89,10 @@ def cache_hash
       def component_routes
```

Comment:
Yes, in the case of a single component the component name is used as title and a link to see_all is displayed. When more than one component is present the title uses the generic name of the component type and the see_all link is disabled because there is no single link for all associated resources.